src/Description.php

```diff
@@ -27,4 +27,9 @@ public static function fromUdb3ModelDescription(Udb3ModelDescription $udb3ModelD
     {
         return new self($udb3ModelDescription->toString());
     }
+
+    public function toUdb3ModelDescription(): Udb3ModelDescription
```

Comment:
Can we use return self?

```suggestion
    public function toUdb3ModelDescription(): self
    {
        return new self($this->toString());
    }
```

Reply:
This class will be deleted https://github.com/cultuurnet/udb3-backend/pull/1821